app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -829,5 +703,217 @@ class BrowserTabFragment : Fragment(), FindListener {
         }
     }
 
+    class BrowserTabFragmentRenderer(private val browserTabFragment: BrowserTabFragment) {
```

Comment:
Do we need this class? I prefer to avoid circular references unless there is a good reason e.g delegate pattern.

Reply:
Will convert to `inner` class and ditch the explicit parameter